src/neoxp/Extensions/Extensions.cs

```diff
@@ -261,5 +262,48 @@ static bool TryGetNEP6Wallet(string path, string password, ProtocolSettings sett
                 }
             }
         }
+        
+        
+        private static JObject[] MakeFindStatesParams(
```

Comment:
I'm aware this function also exists in `StateApi` except that version drops `count` (`pageSize`) if `from` is empty, this version doesn't.

Reply:
Ugh, you're right. Any chance you can open a PR on neo modules to fix it there too? That way we'll eventually be able to remove the express copy of this code.

Reply:
I filed https://github.com/neo-project/neo-express/issues/219 to remind myself to remove the local copy when the updated one in neo-modules is available